### PR TITLE
updated the yaml

### PR DESCRIPTION
**Pull request by mmmiah**
This PR updated the ymal file to document the missing terms in two different models. All other models align within the described model. This PR directly related to #280

### CODE REVIEW

File: transform/models/intermediate/imputation/_imputation.yml [hunk 1]
```diff
@@ -61,13 +61,26 @@ models:
...
+      - name: DETECTOR_IS_GOOD
+        description: Status of the detector based detector's diagonostics.
       - name: VOLUME_SUM
         description: The sum of the flow values for a detector over the sample period.
       - name: OCCUPANCY_AVG
         description: The average of the occupancy values over the sample period.
       - name: SPEED_WEIGHTED
         description: |
           Flow-weighted speed reported by the device.
+      - name: speed_five_mins
```
Comment (kengodleskidot) [inline transform/models/intermediate/imputation/_imputation.yml:81]: @mmmiah we should probably be consistent with either uppercase or lowercase letters for column names in the yml file.

**Review by kengodleskidot — COMMENTED** [review]
Looks good, just a minor formatting comment.

File: transform/models/intermediate/imputation/_imputation.yml (hunk 1 above)
Comment (mmmiah) [reply transform/models/intermediate/imputation/_imputation.yml]: @kengodleskidot , agreed, changed all lowercase column name to uppercase for consistency

**Review by kengodleskidot — COMMENTED** [review]
The updates look good @mmmiah but there is a dbt Cloud check error that needs to be addressed.

**Comment by ian-r-rose** [comment]
> The updates look good @mmmiah but there is a dbt Cloud check error that needs to be addressed.

I'm looking into it in #323, I don't think this PR is the culprit!

**Comment by mmmiah** [comment]
@kengodleskidot @ian-r-rose  same error even after deleting the error file 'ERROR creating sql incremental model dbt_cloud_pr_15_317_clearinghouse.int_clearinghouse__detector_agg_five_minutes  [ERROR in 2369.70s]' for both of my PR. I think that something is wrong with other things.

**Review by ian-r-rose — APPROVED** [review]
This looks good to me, thanks @mmmiah! The CI failure is unrelated, we'll make sure it is fixed and merge this today

**Comment by ian-r-rose** [comment]
Thanks @mmmiah!

**Comment by mmmiah** [comment]
@ian-r-rose , Thanks Ian! Finally it is done

**Pull request merged**